src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task {
```

Comment:
Perhaps put the class in a package?
See [Coding Standard - Package and Import Statements](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements).